src/api/controllers/lines.js

```diff
@@ -231,21 +231,33 @@ function getMyPlaceByEmployerId(user_id, employer_id) {
 }
 
 // get /lines/users?employer_id=xxxxx
+// get /lines/users
 function getUsersByEmployerId(req, res) {
-
-    if (!req.query.employer_id) {
-        res.status(401).json({
+    //if auth user is a recruiter, their emp_id is used (ignore query; recruiters can only see their own batch)
+    //else, use query parameter.
+    //if query blank and auth user isn't a recruiter, not allowed.
+
+    let emp_id;
+    if (req.user.user_type === 'recruiter') {
+        emp_id = req.user.employer_id;
+    } else if (req.query.employer_id) {
+        emp_id = req.query.employer_id;
+    } else {
+        return res.status(401).json({
             "message": response.getLinesUsersMissingEmployerId
         });
-    } else if (req.user.user_type === 'student'){
-        res.status(401).json({
+    }
+
+    if (req.user.user_type === 'student'){
```

Comment:
move this student check above recruiter check.

Reply:
since #134 and #136 are on hold, i'll do this now.